pkg/kv/kvserver/replica_consistency.go

```diff
@@ -783,8 +783,31 @@ A checkpoints directory to aid (expert) debugging should be present in:
 
 A file preventing this node from restarting was placed at:
 %s
+
+Checkpoints are created on each node/store hosting this range, to help
+investigating the cause. Only nodes that are more likely to have incorrect data
+are terminated, usually a majority of replicas continue running. In the most
+serious cases, when there are more than one replica inconsistency, multiple
+nodes can be terminated which may lead to loss of quorum for this replica.
```

Comment:
When does this happen? Will we ever terminate a majority? This sounds edge casey enough that I probably wouldn't mention it, it would only freak out people who are already freaked out.